Energy Saver/Pages/Details.cshtml

```diff
@@ -0,0 +1,54 @@
+﻿@page
+@model Energy_Saver.Pages.DetailsModel
+
+@{
+    ViewData["Title"] = "Details";
+}
+
+<h1 class="text-color">Details</h1>
+
+<div>
+    <hr />
+    <dl class="row">
+        <dt class="col-sm-2 text-color">
+            @Html.DisplayNameFor(model => model.Taxes.Year)
+        </dt>
+        <dd class="col-sm-10 text-color">
+            @Html.DisplayFor(model => model.Taxes.Year)
+        </dd>
+        <dt class="col-sm-2 text-color">
+            @Html.DisplayNameFor(model => model.Taxes.Month)
+        </dt>
+        <dd class="col-sm-10 text-color">
+            @Html.DisplayFor(model => model.Taxes.Month)
+        </dd>
+        <dt class="col-sm-2 text-color">
+            @Html.DisplayNameFor(model => model.Taxes.GasAmount)
```

Comment:
Would be better to display a custom name instead of property name

Reply:
Will fix in the future.